config/routes.rb

```diff
@@ -20,6 +20,8 @@
     post :images, on: :collection
   end
 
+  resources :posts, only: %i[new create show edit update index destroy]
```

Comment:
I think these are all the actions 😅 
You can just do:
```
resources :posts
```

Reply:
They are all the actions yes, I had it like you say at first, but then I had to add this because of the linter: https://github.com/rootstrap/rails_hotwire_playground/runs/2768014626#step:10:42